flight/Libraries/math/misc_math.c

```diff
@@ -255,6 +255,43 @@ void cubic_deadband_setup(float w, float b, float *m, float *r)
 	*r = *m * powf(w, 3) + b * w;
 }
 
+/**
+ * Perform a linear interpolation over N points
```

Comment:
The output range is constrained to [0,1], right? So I couldn't provide a set of ```x``` values with corresponding ```y``` values, and then get the ```y_interpolated``` vs ```x_intermediate``` values.